tests/lemmatizer_sanity_check.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+import sys
+sys.path.append(".")
+from evaluation.lemmatizer_predict import load_and_predict_lemmas
+
+reference_results = pd.read_json('./tests/data/lemmatizer-results.json')
```

Comment:
Add if `__name__ == '__main__'`, we don't want this to be ran if the file is accidentally imported.